### ISSUE TITLE
Noise level of dataset

### ISSUE THREAD

**Issue by Fanke-max** [PeymanMorteza/GEM#1]
Hello! I found the run.sh file set noise level = 0 for odin dataset,  but this is just softmax with temperature. Is the noise level necessary for odin? What's the value of noise for cifar dataset?

**Comment by PeymanMorteza** [comment]
Hello! thanks for your question. For ODIN, "noise" refers to adding small perturbations to the image and can be non-zero. Please see [Enhancing The Reliability of Out-of-distribution Image Detection in Neural Networks](https://arxiv.org/pdf/1706.02690.pdf), Section 3 for more details.
